src/components/app/app-authenticated/app-authenticated.tsx

```diff
@@ -3,7 +3,7 @@
  * See the LICENSE file in the repository root folder for details.
  */
 
-import { Switch, Route } from 'react-router-dom';
+import { Route, Switch } from 'react-router-dom';
```

Comment:
lol this just changed order. try and avoid aimless code changes cuz it just adds confusion imo